src/prefect/agent.py

```diff
@@ -266,7 +266,7 @@ async def check_for_cancelled_flow_runs(self):
         work_pool_filter = (
             WorkPoolFilter(name=WorkPoolFilterName(any_=[self.work_pool_name]))
             if self.work_pool_name
-            else WorkPoolFilter(name=WorkPoolFilterName(any_=["default_agent_pool"]))
+            else WorkPoolFilter(name=WorkPoolFilterName(any_=["default-agent-pool"]))
```

Comment:
When I specified that if a work pool wasn't passed in, the agent should only look for flow runs and queues in `default-agent-pool`, it caused a lot of tests to fail because they were secretly in `test-work-pool` all along, not `default-agent-pool`. This is why I needed to update the tests with a new deployment fixture with no work pool passed in.